docs/discussions.rst

```diff
@@ -208,6 +210,13 @@ Procrastinate will use use one connection to listen to server-side ``NOTIFY`` ca
 :ref:`discussion-general`). The rest of the pool is used for :term:`sub-workers
 <Sub-worker>`.
 
+.. warning::
+
+    Setting ``maxsize`` to ``0`` will actually disable the maximum. Setting it to
```

Comment:
Setting `maxsize` to `0` doesn't really work in aiopg. See https://github.com/aio-libs/aiopg/issues/311. In particular it quickly leads to a `TypeError` when setting `minsize` to `0` as well:
```
>>> pool = await aiopg.create_pool(dsn="", minsize=0, maxsize=0)
>>> await pool._fill_free_pool(True)
Traceback (most recent call last):
  File "/usr/lib64/python3.8/concurrent/futures/_base.py", line 439, in result
    return self.__get_result()
  File "/usr/lib64/python3.8/concurrent/futures/_base.py", line 388, in __get_result
    raise self._exception
  File "<console>", line 1, in <module>
  File "/home/elemoine/.virtualenvs/procrastinate/lib/python3.8/site-packages/aiopg/pool.py", line 208, in _fill_free_pool
    if override_min and self.size < self.maxsize:
TypeError: '<' not supported between instances of 'int' and 'NoneType'
```

And I'm pretty sure `maxsize` set to `0` will eventually cause an error even when `minsize` is not `0`.

So `maxsize=0` is a dangerous setting, and I am not sure we want to advertise it in the Procrastinate documentation.